resources/integration_test/tests/dbmd_constant_result_sets.yml

```diff
@@ -73,53 +73,93 @@ tests:
 
   - description: getTypeInfo_returns_constant_result_set
     db: integration_test
-    skip_reason: "SQL-670: missing null type info in actual result set."
     meta_function: [getTypeInfo]
     expected_result:
-      - [false, false, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         array, false]
-      - [false, false, null, -2, false, null, null, null, 0, 0, 1, 0, 0, 0, 0, 0,
-         binData, false]
-      - [false, false, null, 16, false, null, null, null, 0, 0, 1, 0, 1, 3, 0, 0,
-         bool, false]
-      - [false, false, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         bson, false]
-      - [false, false, null, 93, false, '''', '''', null, 3, 0, 1, 0, 24, 3, 0, 0,
-         date, false]
-      - [false, false, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         dbPointer, false]
-      - [false, false, null, 3, false, null, null, null, 34, 34, 1, 10, 34, 3, 0,
-         0, decimal, false]
-      - [false, false, null, 8, false, null, null, null, 15, 15, 1, 2, 15, 3, 0, 0,
-         double, false]
-      - [false, false, null, 4, false, null, null, null, 0, 0, 1, 2, 10, 3, 0, 0,
-         int, false]
-      - [false, true, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         javascript, false]
-      - [false, true, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         javascriptWithScope, false]
-      - [false, false, null, -5, false, null, null, null, 0, 0, 1, 2, 19, 3, 0, 0,
-         long, false]
-      - [false, false, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         maxKey, false]
-      - [false, false, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         minKey, false]
-      - [false, false, null, 0, false, null, null, null, 0, 0, 1, 0, null, 3, 0, 0,
-         !!str "null", false]
-      - [false, false, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         object, false]
-      - [false, false, null, 1111, false, null, null, null, 0, 0, 1, 0, 24, 3, 0,
-         0, objectId, false]
-      - [false, true, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         regex, false]
-      - [false, true, null, -1, false, '''', '''', null, 0, 0, 1, 0, 0, 3, 0, 0, string,
-         false]
-      - [false, true, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         symbol, false]
-      - [false, false, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         timestamp, false]
-      - [false, false, null, 1111, false, null, null, null, 0, 0, 1, 0, 0, 3, 0, 0,
-         undefined, false]
+      - [binData, -2, null, null, null, null, 1, false, 0, false, null, false, null,
```

Comment:
@bucaojit I had Ethan run the test generator to update this result set. We output all columns lexicographically -- is that ok, or are we supposed to maintain the column ordering described by the javadocs?

Reply:
Thanks for explaining that!